src/libs/events.js

```diff
@@ -20,6 +20,9 @@ const eventsList = {
   cloudEnvironmentCreate: 'cloudEnvironment:create',
   cloudEnvironmentDelete: 'cloudEnvironment:delete',
   cloudEnvironmentUpdate: 'cloudEnvironment:update',
+  catalogueFilter: 'catalogue:filter',
```

Comment:
We've been using "Catalog" instead of "Catalogue" in the ui, can we keep that consistent by changing these?